ansible/playbooks/roles/repository/files/download-requirements/tests/command/debian/test_apt_cache.py

```diff
@@ -1,11 +1,15 @@
+import pytest
 from src.command.debian.apt_cache import AptCache
 
+from tests.data.apt_cache import APT_CACHE_DEPENDS_DATA
 from tests.mocks.command_run_mock import CommandRunMock
 
 
-def test_interface_get_package_dependencies(mocker):
-    ''' Check argument construction for `apt-cache depends` '''
-    with CommandRunMock(mocker, AptCache(1).get_package_dependencies, {'package': 'vim'}) as call_args:
+@pytest.mark.parametrize('PACKAGE_NAME, CMD_STDOUT, EXPECTED_DEPS', APT_CACHE_DEPENDS_DATA)
```

Comment:
Why parametrize a single case?

Reply:
This is to test a few cases (different command's output).